init-tools.sh

```diff
@@ -147,11 +147,11 @@ if [ ! -e $__BUILD_TOOLS_PATH ]; then
 fi
 
 echo "Initializing BuildTools..."
-echo "Running: $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR" >> $__init_tools_log
+echo "Running: $__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR $__PACKAGES_DIR" >> $__init_tools_log
 
 # Executables restored with .NET Core 2.0 do not have executable permission flags. https://github.com/NuGet/Home/issues/4424
 chmod +x $__BUILD_TOOLS_PATH/init-tools.sh
-$__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR >> $__init_tools_log
+$__BUILD_TOOLS_PATH/init-tools.sh $__scriptpath $__DOTNET_CMD $__TOOLRUNTIME_DIR $__PACKAGES_DIR >> $__init_tools_log
```

Comment:
>__PACKAGES_DIR [](start = 82, length = 14)

Should we be keeping our Windows and non-Windows init-tools in sync?

Reply:
For this particular change it doesn't matter much, since Windows is still using full msbuild, but this is definitelly going to be addressed soon once we move to using the CLI to build on Windows as well. However if we do that change now, it might cause some side effects on Windows builds, so I would rather keep this focused to non-Windows for now, given that as I said, the next workitem is to do the same for Windows.